### DISCUSSION TITLE
AUTO CREATION OF EC2 INSTANCE

### DISCUSSION THREAD

**Discussion by devilathag** [learndevopsonline/learndevopsonline#1492] [Q&A]
Hi sir,

i have created only one ec2  ELK instance with instance type r5 large after that i terminate that instance but after few minuets automatically it create another instance with instance type r5 large i am terminating most of instances but again it will recreate new one please find the below screen short and do the needful 

<img width="952" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/144150880/c2bcebe3-a4f6-422f-bb3f-2e54ce3b8ea4">

**Comment by sreedharm07** [comment]
Hi @devilathag 
If you create instance with auto-scaling groups instances will create automatically when you terminate. to avoid this delete auto-scaling groups. then it wont create automatically

**Reply by sreedharm07** [reply]
hi @devilathag
before i  think your error was by auto-scalling , sorry for a mistake.
Since it is a spot instance we created for cost prospective, we need to cancel the spot request, Other wise it will re-create again and again
please cancel spot request to avoid auto creating instance

**Comment by devilathag** [comment]
Dear  Sir,

Still the problem was not resolved pls help me to how to cancel spot
request where i have to cancel this

[image: image.png]


Best regards
G V Devi latha

**Reply by sreedharm07** [reply]
in ec2 service. spot requests. cancel the spot requests which you have
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/7310bc78-2f99-4920-b175-4b1f33519c94)

**Comment by devilathag** [comment]
Thank You so much Sir
